source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -0,0 +1,179 @@
+==============
+Retrieve Data
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to retrieve data from your MongoDB
+database. To retrieve data, use read operations.
+
+Read operations allow you to do the following:
+
+- :ref:`Retrieve a subset of documents from your collection <retrieve-find>`
+- :ref:`Perform transformations on retrieved documents from your collection <retrieve-aggregate>`
+- :ref:`Monitor real-time changes to your database <retrieve-watch>`
+
+Paint Orders
+~~~~~~~~~~~~~
+
+The examples in the following sections are about a paint store that
+keeps track of orders they need to fulfill. The paint store keeps track
+of orders with the ``color`` and ``qty`` fields in their
+``paint_orders`` collection: 
+
+.. code-block:: json
+
+    { "_id": 1, "color": "purple", "qty": 10 }
+    { "_id": 2, "color": "green", "qty": 8 }
+    { "_id": 3, "color": "purple", "qty": 4 }
+    { "_id": 4, "color": "green", "qty": 11 }
+
+Read operations
+---------------
+
+.. _retrieve-find:
+
+Find
+~~~~
+
+Use the find operation to retrieve a subset of your existing data in
+MongoDB. To further specify which documents to retrieve, you can sort,
+skip, and limit the documents or fields.
+
+To perform a find operation, use the ``find()`` method on a
+collection. This method searches a collection for documents that
+match the query filter you provide. For more information on how to
+specify a query, see our :doc:`Specify a Query
+</fundamentals/crud/query-document>` guide. 
+
+To access the results, you can iterate through the results using a
+cursor. For more information on query filters, see our :doc:`Access Data
+From a Cursor </fundamentals/crud/read-operations/cursor>` guide. 
+
+Example
+```````
+
+The paint store would like to know which orders contain more than three, but
+less than ten cans of paint.
+
+To address this scenario, the paint store needs to query the
+``paint_orders`` collection with a filter for orders where their
+quantity is greater than "3", but less than "10" : 
```

Comment:
I think the mix of a high-level description and technical information makes it less easy to read. I wonder if we can separate those out or only present one. If we were just to describe it without technical information, you might say "the paint store needs to find all orders in which the quantity of paint ordered is greater than "3" but less than ten." If you described it with only technical information, you could say, "[the store] builds a query to match all documents in the ``paint_orders`` collection in which the ``qty`` field is greater than "3" and less than "10"."